test/integration/targets/filters/tasks/probe.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Test the probe() filter
```

Comment:
I wonder if adding some negative test cases would also be good, such as undefined value and/or key